spinta/formats/rdf/commands.py

```diff
@@ -0,0 +1,460 @@
+    if '_id' in data:
+        _id = data.pop('_id')
+        if _id.text:
+            attributes[about_name] = get_model_link(model, pk=_id.text)
+    if '_type' in data:
+        _type = data.pop('_type')
+        if _type.text:
+            attributes[type_name] = _type.text
+    return data, attributes
+
+
+def _get_prefix_and_name(uri: str) -> Tuple[str, str]:
+    prefix = name = None
+    if uri and ':' in uri:
+        prefix = uri.split(':')[0]
+        name = uri.split(':')[1]
+    return prefix, name
+
+
+def _create_element(
+    name: Union[str, QName],
+    text: str = None,
+    base: str = None,
+    nsmap: dict = None,
+    attributes: dict = None,
+    children: List[Element] = None,
+) -> Element:
+    if attributes is None:
+        attributes = {}
+    if children is None:
+        children = []
+
+    if nsmap is not None:
+        elem = Element(name, nsmap=nsmap)
+    else:
+        elem = Element(name)
+
+    if base is not None:
+        elem.base = base
+    if text is not None:
+        elem.text = text
+    for key, value in attributes.items():
+        elem.attrib[key] = value
+    for child in children:
+        elem.append(child)
+    return elem
+
+
+def _create_model_element(
+    model: Model,
+    data: dict
+) -> Element:
+    prefixes = _get_available_prefixes(model)
+    about_name = _get_attribute_name('about', RDF, prefixes)
+    type_name = _get_attribute_name('type', RDF, prefixes)
+    model_prefix, model_name = _get_prefix_and_name(model.uri)
+
+    data, attributes = _get_about_and_type_attributes(
+        model,
+        data,
+        about_name,
+        type_name
+    )
+    if model_name:
+        name = _get_attribute_name(model_name, model_prefix, prefixes)
+    else:
+        name = _get_attribute_name(model.basename, model_prefix, prefixes)
+    if not isinstance(name, QName):
+        name = _get_attribute_name(DESCRIPTION, RDF, prefixes)
+        attributes[type_name] = model.model_type()
+
+    return _create_element(
+        name=name,
+        attributes=attributes,
+        children=list(data.values())
+    )
+
+
+@commands.render.register(Context, Request, Model, Rdf)
+def render(
+    context: Context,
+    request: Request,
+    model: Model,
+    fmt: Rdf,
+    *,
+    action: Action,
+    params: UrlParams,
+    data,
+    status_code: int = 200,
+    headers: Optional[dict] = None,
+):
+    headers = headers or {}
+    headers['Content-Disposition'] = f'attachment; filename="{model.basename}.rdf"'
+
+    prefixes = _get_available_prefixes(model)
+    root_name = _get_attribute_name(RDF.capitalize(), RDF, prefixes)
+    root = _create_element(
+        name=root_name,
+        base=str(request.base_url),
+        nsmap=prefixes
+    )
+    if action == Action.GETONE:
+        elem = _create_model_element(model, data)
+        root.append(elem)
+    else:
+        for row in data:
+            elem = _create_model_element(model, row)
+            root.append(elem)
```

Comment:
Could you explain, how this is resolved, I don't see any fix and thing this is still loads all data into memory instead of streaming it to the client, row by row.